views/repo_unavailable.haml

```diff
@@ -0,0 +1,3 @@
+%p
```

Comment:
@tedtoer can't we use existing `404.haml`? What's the point of a new view?

Reply:
@yegor256 we can, but a better choice is to separate behavior of 'not found' and 'unavailable'
And I think its not perfect idea - using 404 always then we have something lost (repo name, tag, log) and will be better to use separate error text for each case

Reply:
@yegor256 but on the other side, unavailable includes 404 case too.

Reply:
@yegor256 If one use not_found for this case (unavailable repo), we have to change behavior of '/log-item' page and use 400-error with human error text, becase it is 400 error when we was not passing tag as GET-parameter etc.

Reply:
@tedtoer make sense. Let's create another page, which will be named `400.haml` and it will just render `e.message` as a text. We will use it for all case where the request is invalid, but not 404.